### PR TITLE
Detail: Animation src-set

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -25,8 +25,19 @@ function scripts_and_styles_method() {
 
     // Then save the gallery size image from each ID into an array
     foreach ($ids as $id) {
```
Comment (mixcas) [inline functions.php]: all of this get images stuff could be a niiiice function. i will fix.

**Review by mixcas — CHANGES_REQUESTED** [review]
code looks alright but it's definitely not working for me. I will check it and fix it.

File: src/js/main.js [hunk 2]
```diff
@@ -83,10 +83,11 @@ Site.HomeAnimation = {
...
 
   addImage: function(imageIndex) {
-    var _this = this;
-
+    var _this = this;    
+    var srcset = _this.generateSrcset(_this.images[imageIndex]);
+    
     // append img tag for image
-    _this.$animation.append('<img src="' + _this.images[imageIndex] + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
+    _this.$animation.append('<img src="' + _this.images[imageIndex]['600'] + '" src-set="' + srcset + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
```
Comment (mixcas) [inline src/js/main.js]: `src-set` is not even a thing…

**Review by mixcas — COMMENTED** [review]
@varffs  did u test ur code?

File: src/js/main.js [hunk 3]
```diff
@@ -83,11 +83,11 @@ Site.HomeAnimation = {
...
 
   addImage: function(imageIndex) {
-    var _this = this;    
+    var _this = this;
     var srcset = _this.generateSrcset(_this.images[imageIndex]);
-    
+
     // append img tag for image
-    _this.$animation.append('<img src="' + _this.images[imageIndex]['600'] + '" src-set="' + srcset + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
+    _this.$animation.append('<img src="' + _this.images[imageIndex]['1800'] + '" srcset="' + srcset + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
```
Comment (varffs) [inline src/js/main.js]: lol I wrote it all and didnt get the attribute name correct. Tho what is best practice for the default size? 1800 is probably too big as the generic fallback

Comment (mixcas) [reply src/js/main.js]: not sure about it 😬

**Pull request merged**